tests/Fixtures/Form/GrossPriceTypeWithPropertyPathType.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the RichModelFormsBundle package.
+ *
+ * (c) Christian Flothmann <christian.flothmann@sensiolabs.de>
+ * (c) Christopher Hertel <christopher.hertel@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace SensioLabs\RichModelForms\Tests\Fixtures\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Exception\InvalidConfigurationException;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class GrossPriceTypeWithPropertyPathType extends AbstractType
```

Comment:
I think I would not add an entirely new form type, but reuse the existing `GrossPriceType` and allow to configure the option there.

Reply:
I could reuse it, but the `property_path` option needs to go into the child field. This means I will have to add an option to the parent form just to pass it along to the child.

Reply:
Yes, I know that sounds a bit confusing at first. But we already have some other fixtures that are built the same. I feel like we should be consistent here.

Reply:
It's removed now